packages/commercetools/theme/nuxt.config.js

```diff
@@ -18,10 +18,15 @@ export default {
       // @todo move to common middleware in core module and cover use case where we don't have slug at all
       // aka URL dispatcher
       routes.push({
-        name: 'custom',
+        name: 'customCategory',
```

Comment:
I think we can just rename those to `category` and `product`, wdyt @EugeneHlushko ?

Reply:
yup makes sense to me, will push an update